### PR TITLE
CLI-530: Fetch vault password from SSM instead of S3

### PR DESCRIPTION
**Pull request by zivshits**
- Tested by deployment and PVWA dashboard

### CODE REVIEW

File: aws/Full-PAS-Deployment.json [hunk 1]
```diff
@@ -3313,8 +3270,8 @@
             "Components":"ami-0c92618583f701176",
             "CPM":"ami-079a10f6ba5c4eeb4",
             "PVWA":"ami-046e947dcae85aafb",
-            "PSM":"ami-0e6d2cea1cf1d0275",
-            "PSMP":"ami-07f89768f972c452d"
+            "PSM":"ami-0ccf83cef896a80ea",
```
Comment (maishsk) [inline aws/Full-PAS-Deployment.json]: Any reason why the AMI Id's changed?

File: aws/PAS-AIO-template.json [hunk 2]
```diff
@@ -1818,7 +1806,7 @@
             "Components":"ami-0c92618583f701176",
             "CPM":"ami-079a10f6ba5c4eeb4",
             "PVWA":"ami-046e947dcae85aafb",
-            "PSM":"ami-0e6d2cea1cf1d0275",
+            "PSM":"ami-0ccf83cef896a80ea",
```
Comment (maishsk) [inline aws/PAS-AIO-template.json]: AMI id changed?

File: aws/Full-PAS-Deployment.json (hunk 1 above)
Comment (zivshits) [reply aws/Full-PAS-Deployment.json]: PSM AMI id points to PSMP AMI. I have fixed only London, but already requested from Nimrod to check all the others

Comment (zivshits) [reply aws/Full-PAS-Deployment.json]: Delete is done as part of "DeletePasswordLambda"

Comment (zivshits) [reply aws/Full-PAS-Deployment.json]: "you removed the S3 access from the policy how are the instances going to get the files from the s3 bucket?" - Except Vault and DR no one require access to S3 bucket. Vault and DR have another policy to access license and recpub. Please see: "VaultInstancesS3VaultFilesBucketPolicy"

File: aws/PAS-AIO-template.json (hunk 2 above)
Comment (zivshits) [reply aws/PAS-AIO-template.json]: See above

File: aws/Full-PAS-Deployment.json (hunk 1 above)
Comment (zivshits) [reply aws/Full-PAS-Deployment.json]: Fixed #2. Consolidated Vault and Component SSM policy

**Pull request merged**